src/uwtools/resources/jsonschema/chgres-cube.jsonschema

```diff
@@ -27,12 +27,57 @@
             "update_values": {
               "properties": {
                 "config": {
-                  "additionalProperties": {
-                    "type": [
-                      "array",
-                      "boolean",
-                      "number",
-                      "string"
+                  "additionalProperties": false,
+                  "anyOf": [
+                    {
+                      "required": [
+                        "data_dir_input_grid",
+                        "atm_files_input_grid"
+                      ]
+                    },
+                    {
+                      "required": [
+                        "data_dir_input_grid",
+                        "grib2_files_input_grid"
+                      ]
+                    },
+                    {
+                      "required": [
+                        "data_dir_input_grid",
+                        "nst_files_input_grid"
+                      ]
+                    },
+                    {
+                      "required": [
+                        "data_dir_input_grid",
+                        "sfc_files_input_grid"
+                      ]
+                    }
+                  ],
+                  "dependencies": {
+                    "atm_files_input_grid": [
+                      "data_dir_input_grid"
+                    ],
+                    "grib2_file_input_grid": [
+                      "data_dir_input_grid"
+                    ],
+                    "nst_files_input_grid": [
+                      "data_dir_input_grid"
+                    ],
+                    "orog_dir_input_grid": [
+                      "orog_files_input_grid"
+                    ],
+                    "orog_dir_target_grid": [
+                      "orog_files_target_grid"
+                    ],
+                    "orog_files_input_grid": [
+                      "orog_files_dir_grid"
+                    ],
+                    "orog_files_target_grid": [
+                      "orog_dir_target_grid"
+                    ],
+                    "sfc_files_input_grid": [
+                      "data_dir_input_grid"
```

Comment:
```suggestion
                     "atm_core_files_input_grid": [
                       "data_dir_input_grid"
                     ],
                     "atm_files_input_grid": [
                       "data_dir_input_grid"
                     ],
                     "atm_tracer_files_input_grid": [
                       "data_dir_input_grid"
                     ],
                     "grib2_file_input_grid": [
                       "data_dir_input_grid"
                     ],
                     "nst_files_input_grid": [
                       "data_dir_input_grid"
                     ],
                     "orog_files_input_grid": [
                       "orog_dir_input_grid"
                     ],
                     "orog_files_target_grid": [
                       "orog_dir_target_grid"
                     ],
                     "sfc_files_input_grid": [
                       "data_dir_input_grid"
                     ],
```


There are some directionality issues in what you have here and some missing options that we'd want to catch.